lib/segment/src/vector_storage/chunked_utils.rs

```diff
@@ -26,7 +27,8 @@ pub fn read_mmaps<T: Sized>(directory: &Path) -> OperationResult<Vec<MmapChunk<T
     for entry in directory.read_dir()? {
         let entry = entry?;
         let path = entry.path();
-        if path.is_file() {
+        if path.is_file() && path.extension().map(OsStr::to_str) != Some(Some(TEMP_FILE_EXTENSION))
```

Comment:
Nit: maybe we need to clean up such resides?

Also, `.map(…) == Some(Some(…))` could be rewritten as `.and_then(…) == Some(…)`.

Reply:
Also, note that later in this code we check that the name conforms the format of `chunk_<number>.mapp`, so checking for `.tmp` is not necessary.

Reply:
Your right, reverted this file changes